plugins/modules/rocketchat.py

```diff
@@ -100,6 +100,14 @@
     elements: dict
     description:
       - Define a list of attachments.
+  is_pre740:
+    description:
+      - If V(true), the payload matches Rocket Chat prior to 7.4.0 format
+      - The default value of the parameter can be set to false in a few months' time
+      - This parameter will be removed in a future release when Rocket Chat 7.4.0 is considered the minimum version to use
```

Comment:
Sorry, but punctuation is important here. This YAML blob is processed and used to generate the module documentation, in this case:
https://docs.ansible.com/ansible/latest/collections/community/general/rocketchat_module.html


```suggestion
      - If V(true), the payload matches Rocket Chat prior to 7.4.0 format.
      - The default value of the parameter can be set to false in a few months' time.
      - This parameter will be removed in a future release when Rocket Chat 7.4.0 becomes the minimum supported version.
```